ee/server/NetworkBroker.ts

```diff
@@ -0,0 +1,225 @@
+import { ServiceBroker, Context, ServiceSchema } from 'moleculer';
+
+import { asyncLocalStorage } from '../../server/sdk';
+import { api } from '../../server/sdk/api';
+import { IBroker, IBrokerNode, IServiceMetrics } from '../../server/sdk/types/IBroker';
+import { ServiceClass } from '../../server/sdk/types/ServiceClass';
+import { EventSignatures } from '../../server/sdk/lib/Events';
+
+const events: { [k: string]: string } = {
+	onNodeConnected: '$node.connected',
+	onNodeUpdated: '$node.updated',
+	onNodeDisconnected: '$node.disconnected',
+};
+
+const lifecycle: { [k: string]: string } = {
+	created: 'created',
+	started: 'started',
+	stopped: 'stopped',
+};
+
+const {
+	INTERNAL_SERVICES_ONLY = 'false',
+	// SERVICES_ALLOWED = '',
```

Comment:
to remove `SERVICES_ALLOWED` you need to add the `internal` flag on every internal service currently missing the flag (like the `TeamService` for example)